trade_remedies_public/templates/v2/registration/registration_check_existing_organisation.html

```diff
@@ -0,0 +1,11 @@
+{% extends "v2/govuk/base_with_form.html" %}
+{% load static %}
+
+{% block page_title %}
+    Existing account for your organisation?
+{% endblock %}
+
+{% block form_content %}
+    {% include 'v2/component_macros/two_radio_buttons.html' with value=radio_value value_one="yes" value_two="no" label_size='xl' label="Has anyone already set up an account for your organisation?" id_one="direct_employee" label_one="Yes" id_two="representative" label_two="No" name="organisation_already_setup" inline="y" %}
```

Comment:
inline should be True so it's clearer to read
```suggestion
    {% include 'v2/component_macros/two_radio_buttons.html' with value=radio_value value_one="yes" value_two="no" label_size='xl' label="Has anyone already set up an account for your organisation?" id_one="direct_employee" label_one="Yes" id_two="representative" label_two="No" name="organisation_already_setup" inline=True %}
```

Reply:
Yes. I tried that, it didn't work - must have used lower case "t".